sdk/storage/azdatalake/internal/generated_blob/zz_models.go

```diff
@@ -554,6 +556,14 @@ type PathItem struct {
 	VersionID  *string            `xml:"VersionId"`
 }
 
+type BlobName struct {
```

Comment:
You'll have to run the `go generate` command to update the auto-generated files.